### PR TITLE
Add Pages test cases 

### PR DESCRIPTION
**Pull request by TelmaRF**
Hi team please review our code
Thanks!

### CODE REVIEW

File: wp_api/tests/pages/tb70_pages.robot
Comment (DenisseC02) [inline wp_api/tests/pages/tb70_pages.robot:1]: Please remove the spanish comment from the line 18

File: wp_api/tests/pages/tb69_pages.robot
Comment (DenisseC02) [inline wp_api/tests/pages/tb69_pages.robot:1]: Please remove the spanish comment from the line 18

Comment (TelmaRF) [reply wp_api/tests/pages/tb69_pages.robot:1]: Thanks! I removed it 😃

File: wp_api/tests/pages/tb70_pages.robot
Comment (Rochius) [inline wp_api/tests/pages/tb70_pages.robot:1]: Hi Telma! The line 18 has a comment in spanish

File: wp_api/tests/pages/tb69_pages.robot
Comment (Rochius) [inline wp_api/tests/pages/tb69_pages.robot:1]: Hi Telma! The line 18 has a comment in spanish

**Review by Rochius — APPROVED** [review]
Looks good to me!

File: wp_api/tests/pages/tb69_pages.robot
Comment (DenisseC02) [reply wp_api/tests/pages/tb69_pages.robot:1]: Perfect, LGTM

**Review by DenisseC02 — APPROVED** [review]

**Pull request merged**